shared-bindings/audiofilters/Filter.c

```diff
@@ -23,7 +23,7 @@
 //|
 //|     def __init__(
 //|         self,
-//|         filter: Optional[synthio.Biquad] = None,
+//|         filters: Optional[List[synthio.Biquad]] = None,
```

Comment:
`filters: tuple[synthio.Biquad, ...] = ()` and similar below

Reply:
See previous comments regarding using `None` or `()` as default.